python/ray/node.py

```diff
@@ -17,6 +17,7 @@
 import ray.ray_constants as ray_constants
 import ray._private.services
 import ray.utils
+import ray.worker
```

Comment:
can you add a short 1 line comment for why this is needed here?

Reply:
This was to get the node ID from the global worker, but unfortunately I think it's not going to work.